typescript/common/data/package.json

```diff
@@ -28,15 +28,15 @@
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/commontoolsinc/labs.git"
+    "url": "git+https://github.com/commontoolsinc/system.git"
   },
   "bugs": {
-    "url": "https://github.com/commontoolsinc/labs/issues"
+    "url": "https://github.com/commontoolsinc/system/issues"
   },
-  "homepage": "https://github.com/commontoolsinc/labs#readme",
+  "homepage": "https://github.com/commontoolsinc/system#readme",
   "dependencies": {},
   "devDependencies": {
-    "@bytecodealliance/jco": "^1.4.0",
+    "@bytecodealliance/jco": "^1.3.1",
```

Comment:
Is this intended to lower the version? Ditto for other package.jsons

Reply:
No, this is an accident; I copied these changes over from another branch-in-progress that I started a while back.